docs/src/tutorials/child-device-config-mngmnt.md

```diff
@@ -0,0 +1,297 @@
+* via HTTP to upload/download files while handling those requests.
+
+For example, during the bootstrapping/startup of the child device, the child device agent needs to upload the supported configuration list of the child device to thin-edge.io by uploading a file using the HTTP file-transfer API of thin-edge.io, followed by an MQTT message informing thin-edge.io that the upload is completed. Similarly, handling a configuration snapshot or update request involves sending MQTT messages before and after the configuration file is uploaded/downloaded via HTTP to/from thin-edge.io.
+
+Since child device agents typically run on an external device and not on the thin-edge.io device itself, the MQTT and HTTP API's of thin-edge.io need to be accessed over the network using its IP address, which is configured using the thin-edge.io configuration settings `mqtt.external.bind_address` or `mqtt.bind_address`. The MQTT APIs are exposed via port 1883 and the HTTP API's are exposed via port 8000. In rare cases, where the child device agent is installed alongside thin-edge.io on the same device, these API's can be accessed via a local IP or even 127.0.0.1.
+
+In this tutorial 127.0.0.1. is used.
+
+##  Prerequisite
+
+To follow this tutorial, you only need the following:
+
+- A [Cumulocity IoT](https://www.softwareag.cloud/site/product/cumulocity-iot.html) tenant.
+- Installed the latest version of thin-edge.io. To this, you can follow [this tutorial](https://thin-edge.github.io/thin-edge.io/html/tutorials/getting-started.html#Step-6-Manage-configuration-files), which will also give you some deeper insights in the various capabilities of thin-edge.io, like configuration management.
+- A Raspberry Pi (3 or 4) with Raspian installed, configured and connected to the Cumulocity tenant. For other boards and OS'es have a look [here](https://github.com/thin-edge/thin-edge.io/blob/main/docs/src/supported-platforms.md)
+- Updated device: `$ sudo apt-get update && sudo apt-get upgrade`
+
+
+## Steps
+
+This tutorial is divided into 3 main steps.
+
+[Step 1 Startup / Bootstrap of the child device](#Startup-/-Bootstrap-of-the-child-device).
+
+In this step a child device with its supported configuration list will be created and will be sent to thin-edge.io during the startup/bootstrap phase of the child device agent.
+
+[Step 2 Handle config snapshot request from thin-edge.io](#handle-config-snapshot-request-from-thin-edge.io)
+
+In this step configuration files from a child device will be requested to make them visible in Cumulocity IoT.
+
+[Step 3 Handle config update request from thin-edge.io](#handle-config-update-request-from-thin-edge.io)
+
+In this configuration files from a child device will be updated.
+
+
+### Step 1 Startup / Bootstrap of the child device
+
+In this step a child device with its supported configuration list will be created and will be sent to thin-edge.io during the startup/bootstrap phase of the child device agent.
```

Comment:
```suggestion
In this step, a child device with its supported configuration list will be created and will be sent to thin-edge.io.
```
We're already in the section titled "Startup / Bootstrap of the child device". So, that second part of the sentence looked redundant.